ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -382,12 +388,14 @@ describe('DataAccessRequirements', () => {
     });
 
     it('should render a mix of complete and incomplete modules, as appropriate', () => {
+        const allModulesSize = allModules.length;
         const incompleteModules = [AccessModule.RASLINKLOGINGOV];
         const completeModules = allModules.filter(module => module !== AccessModule.RASLINKLOGINGOV);
+        const newSize = allModulesSize - incompleteModules.length;
```

Comment:
nit: how about completedSize or simiilar?

Reply:
Done rename to completeModulesSize